api-reference/10 UI Components/dxSlideOut/3 Methods/toggleMenuVisibility(showing).md

```diff
@@ -9,10 +9,10 @@ Shows or hides the slide-out menu depending on the argument.
 A Promise that is resolved after the menu is shown/hidden. It is a <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise" target="_blank">native Promise</a> or a <a href="http://api.jquery.com/Types/#Promise" target="_blank">jQuery.Promise</a> when you use jQuery.
 
 ##### param(showing): Boolean | undefined
-Specifies whether to show or hide the UI component.
+Specifies whether to show or hide the UI component. **undefined** reverses the current state.
 
 ---
-You can use this method instead of [showMenu()](/api-reference/10%20UI%20Components/dxSlideOut/3%20Methods/showMenu().md '/Documentation/ApiReference/UI_Components/dxSlideOut/Methods/#showMenu') and [hideMenu()](/api-reference/10%20UI%20Components/dxSlideOut/3%20Methods/hideMenu().md '/Documentation/ApiReference/UI_Components/dxSlideOut/Methods/#hideMenu') methods. The only difference is that you specify whether to show or hide the UI component passing true or false to the method parameter.
+You can use this method instead of [showMenu()](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Methods/#showMenu) and [hideMenu()](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Methods/#hideMenu) methods. The only difference is that you specify whether to show or hide the UI component when you pass **true**, **false**, or **undefined** to the method parameter.
```

Comment:
```suggestion
You can use this method instead of the [showMenu()]({basewidgetpath}/Methods/#showMenu) and [hideMenu()]({basewidgetpath}/Methods/#hideMenu) methods.
```